File: java/spring-boot/user/src/main/java/com/learnwy/user/com/learnwy/user/dao/UserRepository.java
package com.learnwy.user.com.learnwy.user.dao;

import com.learnwy.user.modal.UserModal;
import org.springframework.data.repository.CrudRepository;

import javax.transaction.Transactional;
import java.util.List;

/**
 * date 2017/9/4
 * create by learnwy
 * lis is GPL-3.0
 */
public interface UserRepository extends CrudRepository<UserModal, Long> {
    List<UserModal> findByUserLoginNameAndUserPassword(String userLoginName,String userPassword);
}

File: kotlin/kotlinSpringBoot/src/main/kotlin/com/learnwy/kotlinSpringBoot/system/repositories/UserRepository.kt
package com.learnwy.kotlinSpringBoot.system.repositories

import com.learnwy.kotlinSpringBoot.system.data.User
import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import org.springframework.data.repository.CrudRepository

interface UserRepository : CrudRepository<User, Long> {
    fun findAll(pageable: Pageable): Page<User>;
}